mds/management/commands/poll_providers.py

```diff
@@ -199,15 +200,27 @@ def get_body(self, provider, url):
     def process_status_changes(self, status_changes, provider):
         if self.verbosity > 1:
             self.stdout.write("Processing...")
+
+        # What is the latest event recorded in that series? (order is not asserted in the specs)
+        try:
+            last_start_time_polled = utils.from_mds_timestamp(
+                max(status_change["event_time"] for status_change in status_changes)
+            )
+        except (KeyError, ValueError):
+            # Data so bad there is no or nothing but invalid event times
+            logger.exception("No valid event_time found in status_changes series: %s", status_changes)
+            # How can we prevent from asking them again next time?
+            return provider.last_start_time_polled + datetime.timedelta(milliseconds=1)
```

Comment:
Why do you need `+ datetime.timedelta(milliseconds=1)` ?

Reply:
I want to avoid polling the same rotten data next time, but I can't know when it is finished.

And you made me realize a mistake...